app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -57,8 +57,8 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
             emergency_banner: document.querySelector('[data-ga4-emergency-banner]') ? 'true' : undefined,
             phase_banner: this.getElementAttribute('data-ga4-phase-banner') || undefined,
             devolved_nations_banner: this.getElementAttribute('data-ga4-devolved-nations-banner') || undefined,
-            cookie_banner: document.querySelector('[data-ga4-cookie-banner]') ? 'true' : undefined,
-            intervention: this.getInterventionPresence(),
+            cookie_banner: this.getBannerPresence('[data-ga4-cookie-banner]'),
```

Comment:
Thanks. I think this potentially broader than this code, as I'm questioning what the point of recording `cookie_banner` in the page view is at all - it will only ever be `true` for the user's first page view, it will always be `true` for the user's first page view, and it will always be `false` after that. I'm questioning why we're making the effort to record something that we always know will be the same. Does that make sense? Is this something we should raise with the analysts?

Reply:
Ah, of course - that's the bit of info I'm missing. I'll make a comment on the card, sorry, not trying to block this, I'll move quickly.